src/clientfactory.ts

```diff
@@ -69,7 +69,7 @@ export class DiscordClientFactory {
             fetchAllMembers: false,
             messageCacheLifetime: 5,
             ws: {
-                intents: Intents.NON_PRIVILEGED,
+                intents: this.config.usePrivilegedIntents ? Intents.ALL : Intents.NON_PRIVILEGED,
```

Comment:
This is typically for usertokens and was purposefully omitted from `this.config.usePrivilegedIntents`

Reply:
The module should have `this.config` at this point, so I guess it shouldn't matter. It's for consistency.

Reply:
That's not the point. The client objects created here are *not* the main bot one. In fact, they are only ever used to send messages, and not read them. They are for if someone has configured a puppet. There is no usecase for priviliged intents for these at all, and them being different tokens does not mean you have priviliged intents enabled for the ones here.

Reply:
should be okay now